main.cpp:13:20: error: invalid conversion from 'EGLNativeDisplayType' {aka 'void*'} to 'Display*' [-fpermissive]
   13 |     Display *dpy = EGL_DEFAULT_DISPLAY;[D

--- src/gui/configure.json.orig	2020-10-27 08:02:11 UTC
+++ src/gui/configure.json
@@ -844,6 +844,7 @@
                     "// has to be disabled in plugins like xcb in this case since the native display,",
                     "// window and pixmap types will be different than what an X-based platform",
                     "// plugin would expect."
+                    , "#define USE_X11"
                 ],
                 "include": [ "EGL/egl.h", "X11/Xlib.h" ],
                 "main": [
